### PR TITLE
Update testdroid proxy with timeout and remove vpn

### PR DESCRIPTION
**Pull request by gregarndt**
Remove the need for providing vpn credentails now that us-west-1 is VPC peered with bitbar.  Also started with a copy of eslintrc from provisioner

### CODE REVIEW

**Comment by gregarndt** [comment]
@jonasfj Mind giving this a quick sanity check?

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "semi": [2, "always"],
+    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+    "default-case": 2,
+    "no-eq-null": 2,
+    "no-floating-decimal": 2,
+    "no-self-compare": 2,
+    "no-throw-literal": 2,
+    "radix": 2,
+    "yoda": [1, "never"],
+    "strict": [2, "global"],
```
Comment (jonasfj) [inline .eslintrc]: No need for `use strict` with babeljs..

Perhaps we need to steal some rules from: https://github.com/babel/babel/blob/master/.eslintrc

Comment (gregarndt) [inline .eslintrc]: ah, I'll update it, this was just a copy from the provisioner I think.

**Comment by jonasfj** [comment]
Looks solid to me.. Feel free to ignore comment on linter, I assume choice of rules is work in progress :)

`r+`, but check that image id doens't need update.. Merge anytime.

**Pull request merged**